build/uuid-redirects.json

```
@@ -53,6 +53,7 @@
     "Compendium.pf2e.classfeatures.Item.S4uY5ap2yhDRqhd0": "Compendium.pf2e.classfeatures.Item.Catharsis Emotion",
     "Compendium.pf2e.classfeatures.Item.T9gUmwghw16itUAO": "Compendium.pf2e.classfeatures.Item.Catharsis Emotion",
     "Compendium.pf2e.classfeatures.Item.zFY2PlJTObXpcQW4": "Compendium.pf2e.classfeatures.Item.Catharsis Emotion",
+    "Compendium.pf2e.deities.Item.liXIbl7y8www2eAn": "Compendium.pf2e.deities.Item.Findeladlara",
```

```suggestion
    "Compendium.pf2e.deities.Item.liXIbl7y8www2eAn": "Compendium.pf2e.deities.Item.Findeladlara",
```
I don't know that we need a redirect for this one.